src/message_buffer.cpp

```diff
@@ -68,46 +66,53 @@ MessageBuffer::push(buffer_type&& data)
 void
 MessageBuffer::pop(size_t length)
 {
-  if (length == 0)
+  if (length == 0) {
     return;
+  }
 
   cleanup(length);
 };
 
 const uint8_t&
 MessageBuffer::front() const
 {
-  if (empty())
+  if (empty()) {
```

Comment:
These braces are better here, though I think the root of the problem is the fact that the clang-format file creates this styling.  What I personally prefer is to keep short `if` statements on a single line.  The syntax I use for that is `AllowShortIfStatementsOnASingleLine: WithoutElse`.

Reply:
Strong disagree.  Too much risk of [`goto fail`](https://www.synopsys.com/blogs/software-security/understanding-apple-goto-fail-vulnerability-2.html) and related bugs.  Note that `AllowShortIfStatementsOnASingleLine: WithoutElse` would not have caught `goto fail`.  Security risk trumps aesthetics.

Reply:
The only risk is when `if` requires more than one line. In this `goto` example, if the `goto` was on the same line as `if`, people would not be adding an extra `goto` lines. The problem is when the code starts looking like Python -- as it did in this case. People then feel compelled to add more lines.